server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/rfc/RequestObjectTest.scala

```diff
@@ -0,0 +1,222 @@
+/** **************************************************************
```

Comment:
No tests for `createdIds` of https://jmap.io/spec-core.html#the-request-object ?